### PR TITLE
Adding Ledger Tendermint provider

### PR DESCRIPTION
**Pull request by jleni**
Renaming/deprecating signatory-cosval.
Adding signatory-ledger-tm
Initial basic benchmarks
Adding provider to README.md

### CODE REVIEW

File: signatory-ledger-tm/Cargo.toml [hunk 1]
```diff
@@ -26,3 +26,11 @@ path = ".."
 
 [dev-dependencies]
 criterion = "0.2"
+
+[features]
+default = []
+#mockhsm = ["ledger-tm/mockhsm"]
```
Comment (Liamsi) [inline signatory-ledger-tm/Cargo.toml]: Do you want to keep this commented out? Maybe delete this line and the empty `default`?

Comment (jleni) [reply signatory-ledger-tm/Cargo.toml]: done

File: README.md [hunk 2]
```diff
@@ -49,27 +50,34 @@ own respective crates:
...
 
 | Provider Crate          | Backend Crate   | Type | Signing | Verification |
-|-------------------------|-----------------|------|---------|--------------|
+| ----------------------- | --------------- | ---- | ------- | ------------ |
 | [signatory‑dalek]       | [ed25519‑dalek] | Soft | 51 k/s  | 18 k/s       |
 | [signatory‑ring]        | [ring]          | Soft | 47 k/s  | 16 k/s       |
 | [signatory‑sodiumoxide] | [sodiumoxide]   | Soft | 38 k/s  | 15 k/s       |
 | [signatory‑yubihsm]     | [yubihsm‑rs]    | Hard | ~8/s    | N/A          |
 
+### Tendermint only providers (amino votes)
```
Comment (Liamsi) [inline README.md]: Not sure people understand what "amino votes" are
```suggestion
### Tendermint only providers (amino encoded consensus votes)
```

File: signatory-ledger-tm/Cargo.toml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 [package]
-name        = "signatory-ledger-cosval"
-description = "Signatory provider for ledger cosmos validator app"
-version     = "0.11.0-pre" # Also update html_root_url in lib.rs when bumping this
+name        = "signatory-ledger-tm"
+description = "Signatory provider for Ledger Tenderming Validator app"
```
Comment (Liamsi) [inline signatory-ledger-tm/Cargo.toml]:
```suggestion
description = "Signatory provider for Ledger Tendermint Validator app"
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -102,13 +102,13 @@ jobs:
...
-        name: signatory-ledger-cosval crate
+        name: signatory-ledger-tm crate
         command: |
           rustc --version
           cargo --version
-          cargo build --package=signatory-ledger-cosval
+          cargo build --package=signatory-ledger-tm
           # Temporarily Disabling testing until a mockhsm is available
-          # cd signatory-ledger-cosval && cargo test
+          # cd signatory-ledger-tm && cargo test
```
Comment (Liamsi) [inline .circleci/config.yml]: Is this tracked somewhere (outside of a comment in the code)? Should I open an issue?

File: signatory-ledger-tm/Cargo.toml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 [package]
-name        = "signatory-ledger-cosval"
-description = "Signatory provider for ledger cosmos validator app"
-version     = "0.11.0-pre" # Also update html_root_url in lib.rs when bumping this
+name        = "signatory-ledger-tm"
+description = "Signatory provider for Ledger Tendermint Validator app"
+version     = "0.11.0" # Also update html_root_url in lib.rs when bumping this
```
Comment (Liamsi) [inline signatory-ledger-tm/Cargo.toml]: looking into lib.rs, it still says 
`html_root_url = "https://docs.rs/signatory-ledger-cosval/0.11.0"` 
Not sure if this also needs renaming.

**Comment by jleni** [comment]
Closing to add support for changes in the Tendermint app

**Pull request closed without merging**